### PR TITLE
Content: rewrite git prep content for week 1

### PR DESCRIPTION
**Pull request by Dedekind561**
## What does this change?

- Updates the prep content for Git Week 1
- Updates the setup instructions for the entire module
- Updates the day-plan

Module: Git
Week(s): 1

### Rewrite sections

- [x] Day plan ( see also CYF-Workshops )
- [x] Prep notes

## Checklist

- [x] I have read the [contributing guidelines](CONTRIBUTING.MD)
- [x] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [x] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [x] I have run my code to check it works
- [x] My changes follow our [Style Guide](https://curriculum.codeyourfuture.io/guides/code-style-guide)

## Description

Add new content for the Git SDC day 1 launch.

### CODE REVIEW

**Comment by SallyMcGrath** [comment]
@Dedekind561 this day is imminent. Please complete this on Tuesday.

**Comment by SallyMcGrath** [comment]
> @SallyMcGrath Does this need to be completed by the end of the day? I thought SDC was launching in Leeds on the 5th August - I was hoping to get this done by the end of the week.

Updating this from our email convo for interested parties:

You are right that the SDC in Birmingham begins on 5 August. As discussed, it's ok to get this done by 24 July. 📆 

We do need to see it before then for review, so please open your PR by 21 July if possible. 🤙 

Thanks for getting in touch! 🙏

File: content/induction/blocks/cyf-folder/index.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+vocabulary=[""]
+emoji= '🧩'
+[objectives]
+1='Navigate and manipulate the file system using a terminal'
+2='Create a cyf directory to store your work over the course'
++++
+
+### Create a cyf folder on your hard drive
+
+If you have not done so already, you'll need to create a CYF folder to store your work on the course.
```
Comment (SallyMcGrath) [inline content/induction/blocks/cyf-folder/index.md:15]:
```suggestion
Create a CYF folder to store your work on the course.
```

File: content/induction/blocks/cyf-folder/index.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+1. Open a terminal on your computer.
+
+For each of the steps below, you'll need to use the command line in your terminal.
+
+You can use this [cli documentation](https://www.techrepublic.com/article/16-terminal-commands-every-user-should-know/) to refresh your memory on terminal commands.
```
Comment (SallyMcGrath) [inline content/induction/blocks/cyf-folder/index.md:21]:
```suggestion
Use this [cli documentation](https://www.techrepublic.com/article/16-terminal-commands-every-user-should-know/) to remember terminal commands.
```

File: content/induction/blocks/cyf-folder/index.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+2. In your terminal, print your current working directory.
+3. List the files and folders in your current working directory.
+
+You'll need a place to store your work for the CYF course.
+
+4. Make a new directory called `cyf` in your home directory.
+5. Change directory into the `cyf` directory.
```
Comment (SallyMcGrath) [inline content/induction/blocks/cyf-folder/index.md:29]:
```suggestion
5. **C**hange **d**irectory into the `cyf` directory.
```

File: content/induction/blocks/cyf-folder/index.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+For each of the steps below, you'll need to use the command line in your terminal.
+
+You can use this [cli documentation](https://www.techrepublic.com/article/16-terminal-commands-every-user-should-know/) to refresh your memory on terminal commands.
+
+2. In your terminal, print your current working directory.
+3. List the files and folders in your current working directory.
+
+You'll need a place to store your work for the CYF course.
+
+4. Make a new directory called `cyf` in your home directory.
```
Comment (SallyMcGrath) [inline content/induction/blocks/cyf-folder/index.md:28]:
```suggestion
4. **M**a**k**e a new **dir**ectory called `cyf` in your home directory.
```

File: content/induction/blocks/cyf-folder/index.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+5. Change directory into the `cyf` directory.
+6. Using the terminal, use a command to create a new directory called `cyf`. You'll use this directory to store the files and folders in your project.
```
Comment (SallyMcGrath) [inline content/induction/blocks/cyf-folder/index.md:30]: Is this step right? It seems like this is step 4 again?

File: content/induction/blocks/github-poets/index.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+time = 15
+facilitation = false
+vocabulary=[""]
+emoji= '🧩'
+[objectives]
+1='Double check Github account is setup'
+2='Double check Course planner is setup'
++++
+
+This playlist will give you an introduction to Git & Github which are two tools we'll be using during the induction week.
```
Comment (SallyMcGrath) [inline content/induction/blocks/github-poets/index.md:13]:
```suggestion
This playlist introduces Git & Github: tools we'll be using from now on.
```

File: content/induction/blocks/github-poets/index.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+These videos cover concepts that you'll be learn slowly over this course as you need them. Be sure to use these videos in the future when you need them.
```
Comment (SallyMcGrath) [inline content/induction/blocks/github-poets/index.md:15]:
```suggestion
These videos cover concepts that you'll learn slowly over this course. Bookmark these videos and come back in the future when you need them.
```

File: content/induction/blocks/github-poets/index.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+
+For now, you only need to watch the first video:
```
Comment (SallyMcGrath) [inline content/induction/blocks/github-poets/index.md:17]:
```suggestion
For now, watch only the first video:
```

File: content/induction/blocks/install-vscode/index.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
++++
+title = '🧰 Install VSCode'
+headless = true
+time = 15
+facilitation = false
+emoji= '🧩'
+[objectives]
+    1='Download and install VSCode'
+    2='Familiarise yourself with VSCode interface'
```
Comment (SallyMcGrath) [inline content/induction/blocks/install-vscode/index.md:9]:
```suggestion
    2='Identify the key parts of the VSCode interface'
```

File: content/induction/blocks/install-vscode/index.md [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
++++
+
+We use VS Code to write all of our code in the course. It is known as an Integrated Developer Environment and really helps you write great code!
```
Comment (SallyMcGrath) [inline content/induction/blocks/install-vscode/index.md:12]:
```suggestion
We use VS Code to write all of our code in the course. It is known as an Integrated Developer Environment (IDE) and really helps you write great code!
```

File: content/induction/blocks/install-vscode/index.md [hunk 11]
```diff
@@ -0,0 +1,17 @@
...
+
+If you've not done so already,
```
Comment (SallyMcGrath) [inline content/induction/blocks/install-vscode/index.md:14]:
```suggestion

```

File: content/induction/blocks/install-vscode/index.md [hunk 12]
```diff
@@ -0,0 +1,17 @@
...
+please follow the link below and download the appropriate version for your operating system.
```
Comment (SallyMcGrath) [inline content/induction/blocks/install-vscode/index.md:15]:
```suggestion

```

File: content/induction/blocks/install-vscode/index.md [hunk 13]
```diff
@@ -0,0 +1,17 @@
...
+
+[🔗 Download and install VSCode here](https://code.visualstudio.com/)
```
Comment (SallyMcGrath) [inline content/induction/blocks/install-vscode/index.md:17]:
```suggestion
[🔗 Download and install VSCode now](https://code.visualstudio.com/)
```

File: content/induction/blocks/learning-as-community/index.md [hunk 14]
```diff
@@ -0,0 +1,30 @@
...
+    2='Describe a typical week at Code Your Future'
++++
+
+At Code Your Future, we learn as a **community.**
+
+In this session, we're going to describe how we harness our community to overcome obstacles and develop our understanding.
+
+### Resources
+
+The facilitator will need to use this presentation 👉 [Learning as a community presentation](https://miro.com/app/board/uXjVPy92Cu8=/)
```
Comment (SallyMcGrath) [inline content/induction/blocks/learning-as-community/index.md:18]:
```suggestion
The facilitator will use this presentation 👉 [Learning as a community presentation](https://miro.com/app/board/uXjVPy92Cu8=/)
```

File: content/induction/blocks/learning-as-community/index.md [hunk 15]
```diff
@@ -0,0 +1,30 @@
...
+
+### Preparation
+
+- [ ] Facilitator: Review this [presentation](https://miro.com/app/board/uXjVPy92Cu8=/) before class.
+- [ ] Facilitator: Ensure everyone can access the Miro board above.
```
Comment (SallyMcGrath) [inline content/induction/blocks/learning-as-community/index.md:23]:
```suggestion
- [ ] Facilitator: Ensure everyone can access the Miro board presentation.
```

File: content/induction/blocks/learning-as-community/index.md [hunk 16]
```diff
@@ -0,0 +1,30 @@
...
+- [ ] Facilitator: Make sure everyone is split into groups of no more than 5.
```
Comment (SallyMcGrath) [inline content/induction/blocks/learning-as-community/index.md:24]:
```suggestion
- [ ] Facilitator: Split the class into groups of no more than 5.
```

File: content/induction/blocks/learning-as-community/index.md [hunk 17]
```diff
@@ -0,0 +1,30 @@
...
+- [ ] Facilitator: Make sure every group has access to a laptop.
+- [ ] Facilitator: Make sure every group has access to a piece of paper and pen
+
+### Introduction
+
+The facilitator can use the Miro board presentation to guide trainees and volunteers on a discussion about how we learn together as a community.
```
Comment (SallyMcGrath) [inline content/induction/blocks/learning-as-community/index.md:30]:
```suggestion
The facilitator will use the Miro board presentation to guide trainees and volunteers on a discussion about how we learn together as a community.
```

Comment (SallyMcGrath) [reply content/induction/blocks/learning-as-community/index.md:30]: I suggest you link them to a recording of you doing this if possible, to help the volunteer know what is expected.

File: content/induction/blocks/next-steps/index.md [hunk 18]
```diff
@@ -0,0 +1,27 @@
++++
+title = 'Next steps 👣'
+headless = true
+time ="30"
+facilitation = false
+emoji= '🧩'
+[objectives]
+    1='Figure out what we need to do next in the course'
+    2='Familiarise ourselves with the curriculum'
```
Comment (SallyMcGrath) [inline content/induction/blocks/next-steps/index.md:9]:
```suggestion
    2='Locate the prep view on the curriculum'
```

Can you rephrase this as a testable act?

File: content/induction/blocks/next-steps/index.md [hunk 19]
```diff
@@ -0,0 +1,27 @@
...
+## For trainees
+
+In this section, we're going to go through a set of questions/tasks to clarify what we're going to do in the next part of the course.
+
+Get into pairs with somebody you've not paired up with today and try figuring out the answers/solutions to the following questions/tasks:
+
+1. Using the [curriculum website](https://curriculum.codeyourfuture.io/), find the page with the prep material for the upcoming module
+2. List the things you need to install ( if anything ) for the upcoming module.
+3. Check you can find the backlog page for the upcoming module.
+4. Go to your Slack helpdesk channel and start a thread 🧵 outlining times you're free to meet up to discuss the prep work during the week
```
Comment (SallyMcGrath) [inline content/induction/blocks/next-steps/index.md:22]:
```suggestion
4. Go to your group Slack channel and start a thread 🧵 outlining times you're free to meet up to discuss the prep work during the week
```

They might not have a helpdesk channel - not all regions do.

File: content/induction/sprints/1/prep/index.md [hunk 20]
```diff
@@ -0,0 +1,353 @@
...
+description = 'You can use these notes to prepare for the first in-person session of the software development course'
+layout = 'prep'
+emoji= '📝'
+menu_level = ['sprint']
+weight = 1
+backlog= 'Module-Induction'
+backlog_filter= 'Induction'
++++
+
+## Learning objectives
```
Comment (SallyMcGrath) [inline content/induction/sprints/1/prep/index.md:12]: Can you list these in the front matter so they can be picked up by other views, please?

Same setup as a block/index.md

File: content/induction/sprints/1/prep/index.md [hunk 21]
```diff
@@ -0,0 +1,353 @@
...
+Given a remote or local repository, identify the number of commits
+Given a remote or local repository, identify the author, time and message of a given commit
+Given a remote or local repository, identify the files and folders from any commit in a version timeline
+Describe the purpose of version control
+Explain the difference between Git and Github
+Get a local copy of a repository on your local machine
+
+## 📝 CYF blog
+
+Imagine a scenario where 2 people are writing a blog.
```
Comment (SallyMcGrath) [inline content/induction/sprints/1/prep/index.md:25]:
```suggestion
Imagine that two people are writing a blog.
```

File: content/induction/sprints/1/prep/index.md [hunk 22]
```diff
@@ -0,0 +1,353 @@
...
+Ahmed and Naima are co-authoring a blog to document the things they're struggling with whilst learning to code.
```
Comment (SallyMcGrath) [inline content/induction/sprints/1/prep/index.md:26]:
```suggestion
Ahmed and Naima are writing a blog together about their struggles learning coding.
```

File: content/induction/sprints/1/prep/index.md [hunk 23]
```diff
@@ -0,0 +1,353 @@
...
+
+The published blog is live here 👉 [CYF educational blog](https://git-demo-week1.netlify.app/)
+
+{{<note type="exercise" title="exercise 1.1">}}
+Try exploring the site above. Try clicking on the link on the site and check to see if its working.
```
Comment (SallyMcGrath) [inline content/induction/sprints/1/prep/index.md:31]:
```suggestion
Explore the [CYF educational blog](https://git-demo-week1.netlify.app/). Click on the link on the site and check to see if it is working.
```

File: content/induction/sprints/1/prep/index.md [hunk 24]
```diff
@@ -0,0 +1,353 @@
...
+{{</note>}}
+
+## 🧰 Development process
+
+Ahmed and Naima are currently using the following **development process** for writing the blog:
```
Comment (SallyMcGrath) [inline content/induction/sprints/1/prep/index.md:36]:
```suggestion
Ahmed and Naima are using the following **development process** for writing their blog:
```

File: content/induction/sprints/1/prep/index.md [hunk 25]
```diff
@@ -0,0 +1,353 @@
...
+
+> - writing the blog in a single file on a **single** computer
+> - saving multiple versions of the file on the same computer
+> - they take it in turns to the use the computer at different points in the day
```
Comment (SallyMcGrath) [inline content/induction/sprints/1/prep/index.md:40]:
```suggestion
> - taking turns to use the computer during the day
```

File: content/induction/sprints/1/prep/index.md [hunk 26]
```diff
@@ -0,0 +1,353 @@
...
+
+At the moment, the computer has a folder with the blog that looks like this:
+
+![different-blog-versions](different-blog-versions.png)
+
+{{<note type="exercise" title="exercise 1.2">}}
+
+Describe some of the challenges that Ahmed and Naima face when trying to write a blog together in this way.
+
+Try creating a thread 🧵 Slack to see if your cohort colleagues came up with similar answers.
```
Comment (SallyMcGrath) [inline content/induction/sprints/1/prep/index.md:50]:
```suggestion
Create or join a thread 🧵 on Slack to share your answers. Did your cohort colleagues think of similar things?
```

File: content/induction/sprints/1/prep/index.md [hunk 27]
```diff
@@ -0,0 +1,353 @@
...
+
+**Github** is a platform where teams can store projects along with a history of their different versions.
+
+By storing projects on Github, multiple users can gain access to the history of a project.
+
+{{<note type="definition" title="Definition">}}
+On Github we call our project and its history a **repository**.
+{{</note>}}
+
+{{<note type="exercise" title="Exercise 2.1">}}
```
Comment (SallyMcGrath) [inline content/induction/sprints/1/prep/index.md:100]: Could we have more meaningful titles please? To create a better navigation experience.

File: content/induction/sprints/1/prep/index.md [hunk 28]
```diff
@@ -0,0 +1,353 @@
...
+
+{{</note>}}
+
+{{<note type="exercise" title="exercise 2.3">}}
+
+### Explore 🔍
+
+Go to the following link: https://github.com/CodeYourFuture/education-blog/commits/main
+
+Try answering the following questions:
```
Comment (SallyMcGrath) [inline content/induction/sprints/1/prep/index.md:151]:
```suggestion
#### Questions:
```

File: content/induction/sprints/1/prep/index.md [hunk 29]
```diff
@@ -0,0 +1,353 @@
...
+
+Go to the commit that says "add test p element to index page"
+
+Try answering the following questions:
```
Comment (SallyMcGrath) [inline content/induction/sprints/1/prep/index.md:137]:
```suggestion
#### Questions
```

File: content/induction/sprints/1/prep/index.md [hunk 30]
```diff
@@ -0,0 +1,353 @@
...
+
+### Explore 🔍
+
+Go to the following link: https://github.com/CodeYourFuture/education-blog/commits/main
+
+Try answering the following questions:
+
+Go to the commit that says "remove \ and # from start of paragraph"
+
+Try answering the following questions:
```
Comment (SallyMcGrath) [inline content/induction/sprints/1/prep/index.md:155]:
```suggestion
#### Questions:
```

File: content/induction/sprints/1/prep/index.md [hunk 31]
```diff
@@ -0,0 +1,353 @@
...
+- How many files were changed in this commit?
+- What change was made in this commit?
+
+{{</note>}}
+
+## 📜 Previous versions
+
+We can view the different commits of a project on Github.
+
+Therefore we can also see what the website looked at previous versions too.
```
Comment (SallyMcGrath) [inline content/induction/sprints/1/prep/index.md:166]:
```suggestion
This means we can see what the website looked like before, in previous versions.
```

File: content/induction/sprints/1/prep/index.md [hunk 32]
```diff
@@ -0,0 +1,353 @@
...
+Version B should have nothing on the index page
+-->
+
+[Deployed version C educational blog](https://64b971b0adae9e00089d7531--git-demo-week1.netlify.app/)
+
+<!---
+Version C should be same as production deployment but with some rogue characters on the page
+-->
+
+Answer the following questions:
```
Comment (SallyMcGrath) [inline content/induction/sprints/1/prep/index.md:190]:
```suggestion
#### Questions:
```

File: content/induction/blocks/cyf-folder/index.md [hunk 33]
```diff
@@ -0,0 +1,31 @@
++++
+title = '🧰 Create a CYF folder'
```
Comment (SallyMcGrath) [inline content/induction/blocks/cyf-folder/index.md:2]: There's a video for this here https://www.youtube.com/watch?v=JFX8UfXWmok

You might want to pop the video in a tab?

File: content/induction/sprints/1/prep/index.md [hunk 34]
```diff
@@ -0,0 +1,353 @@
...
+
+Follow through the steps in this video 👉 https://www.youtube.com/watch?v=ILJ4dfOL7zs ( Remember to use the url for the education-blog repo when you are cloning )
+
+{{</note>}}
+
+## 📘 Viewing the files
+
+Once you've got a local copy of a codebase on your local machine you can start to view the files and folders in that codebase. You can use a code editor like VSCode.
+
+VSCode is an application that enables developers to view and edit files on their local machine.
```
Comment (SallyMcGrath) [inline content/induction/sprints/1/prep/index.md:276]:
```suggestion
VSCode is an application developers use to view and edit files on their local machine.
```

File: content/induction/sprints/1/prep/index.md [hunk 35]
```diff
@@ -0,0 +1,353 @@
...
+
+You'll need to clone the following repository 👉 https://github.com/CodeYourFuture/education-blog
+
+Follow through the steps in this video 👉 https://www.youtube.com/watch?v=ILJ4dfOL7zs ( Remember to use the url for the education-blog repo when you are cloning )
+
+{{</note>}}
+
+## 📘 Viewing the files
+
+Once you've got a local copy of a codebase on your local machine you can start to view the files and folders in that codebase. You can use a code editor like VSCode.
```
Comment (SallyMcGrath) [inline content/induction/sprints/1/prep/index.md:274]:
```suggestion
Once you've got a local copy of a codebase on your local machine you can start to view the files and folders in that codebase.  Use VSCode.
```

File: content/induction/sprints/1/prep/index.md [hunk 36]
```diff
@@ -0,0 +1,353 @@
...
+
+{{</note>}}
+
+## 🌳 Branching
+
+We can check the commits on the remote repository as before:
+
+![commit-history](commit-history.png)
+
+On the left page of the page, we see additional information:
```
Comment (SallyMcGrath) [inline content/induction/sprints/1/prep/index.md:296]:
```suggestion
On the left page of the page, we see morel information:
```

File: layouts/partials/block/data.html [hunk 37]
```diff
@@ -52,6 +52,7 @@
     */}}
     {{ $newSrc = replace $newSrc "/tree/main" "/readme" }}
     {{ $newSrc = replace $newSrc "/blob/main/README.md" "/readme" }}
+    {{ $newSrc = "https://api.github.com/repos/CodeYourFuture/CYF-Workshops/readme/git-day-1?ref=rewrite/git-day-1" }}
```
Comment (SallyMcGrath) [inline layouts/partials/block/data.html:55]:
```suggestion
    {{ $newSrc = "https://api.github.com/repos/CodeYourFuture/CYF-Workshops/readme/git-day-1?ref=rewrite/git-day-1" }}
```

Looks like you've hardcoded in a single source, which doesn't seem desirable. ;P 
```suggestion

```

File: layouts/shortcodes/details.html [hunk 38]
```diff
@@ -1,4 +1,4 @@
 <details>
-  <summary><h2>{{ $.Params.summary }}</h2></summary>
+  <summary><h3>{{ $.Params.summary }}</h3></summary>
```
Comment (SallyMcGrath) [inline layouts/shortcodes/details.html:2]: Please move out all code changes into a new PR and have this as content only. Thanks!

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]
Done a quick pass. Will do another after changes are made as it will get too overwhelming otherwise I think.

Thanks for your hard work on this

File: content/induction/blocks/cyf-folder/index.md [hunk 39]
```diff
@@ -0,0 +1,43 @@
...
+4. **M**a**k**e a new **dir**ectory called `cyf` in your home directory.
+
+5. **C**hange **d**irectory into the `cyf` directory.
+
+6. Double check you're in the right place by **p**rinting your current **w**orking **d**irectory.
+
+{{% /tab %}}
+
+{{<tab name="📼 Watch: How to create a CYF folder">}}
+{{% youtube %}}https://www.youtube.com/watch?v=JFX8UfXWmok{{% /youtube %}}
```
Comment (Dedekind561) [inline content/induction/blocks/cyf-folder/index.md:40]: This video isn't being rendered in the UI - same is true for this tab in production too. https://curriculum.codeyourfuture.io/fundamentals/prep/

File: content/induction/sprints/1/day-plan/index.md [hunk 40]
```diff
@@ -2,14 +2,32 @@
...
+time="15"
+[[blocks]]
+name="learning-as-community"
+src="module"
+[[blocks]]
+name="lunch"
+[[blocks]]
+name="Git workshop"
+src="https://github.com/CodeYourFuture/CYF-Workshops/blob/rewrite/git-day-1/git-day-1/README.md"
+time="120"
```
Comment (Dedekind561) [inline content/induction/sprints/1/day-plan/index.md:27]: I can't update the time for this block "Git workshop" - not sure what I'm doing wrong here. 🤔

**Review by illicitonion — COMMENTED** [review]
This is generally looking great! Really high quality stuff!

Other than the specific line comments, one more thing:

The workshop asks questions early on about a "hash" of a commit, but we don't introduce that anywhere in the prep. The diagrams in the prep page also refer to hashes, but don't explain them, which is probably confusing.

We should probably introduce the term, and vaguely what the concept is, maybe alongside the first commit history mermaid diagram?

File: content/induction/blocks/cyf-folder/index.md (hunk 39 above)
[the remark this answers is not in the document]
Comment (Dedekind561) [reply content/induction/blocks/cyf-folder/index.md:40]: To be honest I didn't expect the video in great depth - I'm happy to leave at this stage. I think one set of clear instructions is preferable at this point.

File: content/induction/sprints/1/prep/index.md [hunk 41]
```diff
@@ -0,0 +1,358 @@
...
+
+You'll learn more about these type of files throughout the course.
+
+{{</note>}}
+
+> We can use the Github interface to explore the different **commits** ( versions ) of a project too.
+
+{{<note type="exercise" title="exercise 2.2">}}
+
+### Explore 🔍
```
[the remark this answers is not in the document]
Comment (Dedekind561) [reply content/induction/sprints/1/prep/index.md:134]: I actually did have this section in the original draft and then took it out - I do agree I think it would be super useful for them to browse the repository somewhere in the past!
I'll reinstate that section.

File: content/induction/sprints/1/prep/index.md [hunk 42]
```diff
@@ -0,0 +1,358 @@
...
+
+{{<note type="exercise" title="Exercise 5.1 - 👣 Follow the steps">}}
+
+###
+
+> 🎯 Goal: Clone a remote repository to a local machine
+
+You'll need to use the information below to achieve this goal:
+
+You'll need to clone the following repository 👉 https://github.com/CodeYourFuture/education-blog
```
[the remark this answers is not in the document]
Comment (Dedekind561) [reply content/induction/sprints/1/prep/index.md:271]: They need to clone their fork here - I'll make that clearer

File: content/induction/blocks/learning-as-community/index.md [hunk 43]
```diff
@@ -0,0 +1,30 @@
...
+    2='Describe a typical week at Code Your Future'
++++
+
+At Code Your Future, we learn as a **community.**
+
+In this session, we're going to describe how we harness our community to learn together.
+
+### Resources
+
+The facilitator will use this presentation 👉 [Learning as a community presentation](https://miro.com/app/board/uXjVPy92Cu8=/)
```
[the remark this answers is not in the document]
Comment (Dedekind561) [reply content/induction/blocks/learning-as-community/index.md:18]: I've just not published it yet so I'll fix the permissions on that once I've spruced it up

File: content/induction/sprints/1/prep/index.md [hunk 44]
```diff
@@ -0,0 +1,358 @@
...
+        commit
+        commit
+        commit
+```
+
+In the diagram above, we can continue to commit on the "week-1-coursework" branch without altering the history of the `main` branch.
+
+{{<note type="exercise" title="exercise 6.1">}}
+
+### Creating a local branch
```
[the remark this answers is not in the document]
Comment (Dedekind561) [reply content/induction/sprints/1/prep/index.md:360]: Cool, I did wonder about this - I think I'll link them to a video reference they can use to try and figure this out. If they don't quite get it in then it will be covered in the Git workshop on Saturday.

File: content/induction/sprints/1/prep/index.md [hunk 45]
```diff
@@ -0,0 +1,358 @@
...
+Version B should have nothing on the index page
+-->
+
+[Deployed version C educational blog](https://64b971b0adae9e00089d7531--git-demo-week1.netlify.app/)
+
+<!---
+Version C should be same as production deployment but with some rogue characters on the page
+-->
+
+#### Questions:
```
[the remark this answers is not in the document]
Comment (Dedekind561) [reply content/induction/sprints/1/prep/index.md:214]: A combination of the 2 strategies I think

**Comment by Dedekind561** [comment]
@illicitonion I've just been through your comments & suggestions - all super useful, thankyou! I've addressed all of them bar this: https://github.com/CodeYourFuture/curriculum/pull/168#discussion_r1275467378. Are you happy for this to go live and then we can fix this in another PR? I just want to share with West Midlands volunteers asap.
